### ISSUE TITLE
Leviathan Driver Turret

### ISSUE THREAD

**Issue by HellDragon-HK** [GreatEmerald/UT3Vehicles#120]
- ANIMATION RELATED
  - [ ] Driver turret when entering main cannon mode in UT3 always seems to rotate itself so the barrels are toward the main cannon with the back of the turret coming down the front, currently
  - [X] Driver Turret needs to be scaled up,  X = 1.7  Y = 1.55 or 1.7  Z = 1.5 looks fairly close

**Comment by GreatEmerald** [comment]
Yes, the projectiles in UT3 home as well. I'm not sure if they home quite as much, however.

**Comment by HellDragon-HK** [comment]
Ok let me see if I understand this, by the look of it they are set to not so much home as be forcefully directed to the target if they have more than 1.3 lifespan remaining, is that right?

http://greatemerald.xmpcommunity.com/uncodex/UT3-15/Source_utgamecontent/utproj_leviathanbolt.html#30

**Comment by GreatEmerald** [comment]
Nah, they home if they have more than 3.5 lifespan remaining. The changes are to acceleration, so they are gradual. Force in this case probably means attraction force.

You can see that the same code is in UT2004:
http://greatemerald.xmpcommunity.com/uncodex/UT2004/Source_onslaughtfull/onsmasrocketprojectile.html#49
Except it has three differences: the static number is different, acceleration is summed in UT3, and it stops when the LifeSpan hits 3.5. In other words, looks like it homes more in UT3, but only initially.

**Comment by HellDragon-HK** [comment]
So any changes needed there or we leaving it as it?

**Comment by GreatEmerald** [comment]
These changes are significant, so they should be ported. It's also pretty easy to do so.
